lib/config/presets/internal/regex-managers.ts

```diff
@@ -16,10 +16,10 @@ export const presets: Record<string, Preset> = {
   },
   githubActionsVersions: {
     description:
-      'Update `_VERSION` environment variables in GitHub Action files.',
+      'Update `_VERSION` environment variables in GitHub Actions files.',
```

Comment:
```suggestion
      'Update `_VERSION` environment variables in GitHub Action files.',
```
looks wrong too me

Reply:
Reverted back in https://github.com/renovatebot/renovate/pull/19966/commits/c3fa63ed60f0beb2a97c23ce7ac7084cf84d35fd.